### PR TITLE
Use an array instead of switch for printing colors

### PR DESCRIPTION
**Pull request by suve**
This changeset adds two arrays, `fcstyle` and `bcstyle`, holding information about foreground and background colours. This allows us to remove the colouring `switch` statement and replace it with a simple `printf(array[index])`.

### CODE REVIEW

File: aha.c [hunk 1]
```diff
@@ -360,7 +386,30 @@ int main(int argc,char* args[])
 			printf(".bold        {font-weight: bold;}\n");
 			printf(".blink       {text-decoration: blink;}\n");
 			printf("</style>\n");
+			
+			fcstyle[0] = "dimgray ";
+			fcstyle[1] = "red ";
+			fcstyle[2] = "green ";
+			fcstyle[3] = "yellow ";
+			fcstyle[4] = "blue";
```
Comment (theZiz) [inline aha.c]: You forgot the space after `blue`

**Comment by suve** [comment]
I will need to update this PR to work properly with the header printing moved to a separate function.

**Comment by theZiz** [comment]
Well, up to you. I would have merged it by hand. :wink:

**Comment by suve** [comment]
Thanks for merging! Sorry for not resolving conflicts like I said earlier, I was a tad busy lately.

**Comment by theZiz** [comment]
No problem, I myself needed a long time to merge as you noticed... :laughing:  Thanks a lot for your work, I really approciate your changes. :+1: While extending `aha` I always noticed, that I should have done things differently, but I was too lazy and just hacked more into it. Your refactoring was really needed! :tada:

**Pull request merged**